[17-49-44] [EXCEPTION]:
Exception origin: cyder.handlers.internal.Logger.fixLogs(Logger.java:577)
From line: 577
Trace: java.lang.NullPointerException
at cyder.handlers.internal.Logger.fixLogs(Logger.java:577)
at cyder.handlers.internal.Logger.initialize(Logger.java:277)
at cyder.genesis.Cyder.main(Cyder.java:50)
[17-49-44] [SYSTEM_IO]: System data pared in IOUtil's static block
[17-49-44] [SYSTEM_IO]: Suggestions pared in IOUtil's static block
[17-49-44] [SYSTEM_IO]: Exit conditions pared in IOUtil's static block
[17-49-44] [SYSTEM_IO]: DebugHashes pared in IOUtil's static block
[17-53-08] [EOL]: Log completed, Cyder crashed unexpectedly: exit code: -200 [External Stop], exceptions thrown: 1
